tests/Integration/goldens/compute_small/src/V1/Gapic/AddressesGapicClient.php

```diff
@@ -142,6 +147,35 @@ private static function getSupportedTransports()
         ];
     }
 
+    /**
+     * Return an RegionOperationsClient object with the same endpoint as $this.
+     *
+     * @return RegionOperationsClient
+     */
+    public function getOperationsClient()
+    {
+        return $this->operationsClient;
+    }
+
+    /**
+     * Resume an existing long running operation that was previously started by a long
+     * running API method. If $methodName is not provided, or does not match a long
+     * running API method, then the operation can still be resumed, but the
+     * OperationResponse object will not deserialize the final response.
+     *
+     * @param string $operationName The name of the long running operation
+     * @param string $methodName    The name of the method used to start the operation
+     *
+     * @return OperationResponse
+     */
+    public function resumeOperation($operationName, $methodName = null)
+    {
+        $options = isset($this->descriptors[$methodName]['longRunning']) ? $this->descriptors[$methodName]['longRunning'] : [];
+        $operation = new OperationResponse($operationName, $this->getOperationsClient(), $options);
```

Comment:
Ok after talking offline, these are the default descriptors we are currently generating for the DIREGAPIC LRO methods. The solution we are going to try is generating a private method (only for DIREGAPIC clients) that supplies the default descriptors for an Operation Service in case the `methodName` is not supplied here or it does not appear in the descriptor set. Will update when I've implemented it.

Reply:
Ok, I added generation of a `getDefaultOperationDescriptor` method that returns the descriptor array from one of the methods in the service that utilizes the custom operation.